tensorflow/core/kernels/conv_ops_gpu_3.cu.cc

```diff
@@ -640,15 +641,12 @@ struct BatchNarrowMatrixTransposeDispatcher<
   }
 };
 
-template <typename T, int TileLongSide, int TileShortSide>
-struct BatchNarrowMatrixTransposeDispatcher<
-    T, TileLongSide, TileShortSide,
-    typename std::enable_if<TileSizeOutsideBoundary(TileLongSide, TileShortSide,
-                                                    sizeof(T)),
-                            void>::type> {};
-
 // This function tries to recover, in a brute force way, the frontier defined in
 // TileSizePossibilityFrontierCheck as a vector of tile size combinations.
+// Note that if one changes the frontier definition as in
+// TileSizePossibilityFrontierCheck and forgets to set the corresponding short
+// side len of the last legal long side len with just one possibility of 2, this
+// function will fail and produce an error.
```

Comment:
Doesn't really produce an error, but rather crashes the program (or "will raise a fatal error", if you prefer).